discussment-core/src/test/resources/META-INF/persistence.xml

```diff
@@ -19,7 +19,7 @@
             <property name="hibernate.show_sql" value="false" />
             <property name="hibernate.format_sql" value="true" />
             <property name="hibernate.dialect" value="org.hibernate.dialect.MySQL5InnoDBDialect" />
-            <!--<property name="hibernate.hbm2ddl.auto" value="create" />-->
+            <property name="hibernate.hbm2ddl.auto" value="validate"/>
             <!--<property name="hibernate.hbm2ddl.import_files" value="/META-INF/sql/schema.sql, /META-INF/sql/data.sql" />-->
```

Comment:
Are those sql scripts important (e.g. test data)? If so I would keep the commented line. Otherwise, I think it should be deleted.

Reply:
Nope, they are not used. All unit tests are using mocks now and integration tests module has own sql script picked by Flyway.